java-extension/com.microsoft.java.test.plugin/src/main/java/com/microsoft/java/test/plugin/util/TestSearchUtils.java

```diff
@@ -111,12 +112,16 @@
     }
 
     public static ASTNode parseToAst(final ICompilationUnit unit, IProgressMonitor monitor) {
-        final ASTParser parser = ASTParser.newParser(AST.JLS14);
-        parser.setSource(unit);
-        parser.setFocalPosition(0);
-        parser.setResolveBindings(true);
-        parser.setIgnoreMethodBodies(true);
-        return parser.createAST(monitor);
+        final CompilationUnit astRoot = CoreASTProvider.getInstance().getAST(unit, CoreASTProvider.WAIT_YES, monitor);
```

Comment:
is there any difference if setting to WAIT_NO?

Reply:
Check and not very significant difference observed